.github/workflows/develop.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - "develop"
       - "release-*"
+      - "TASK-*"
```

Comment:
This is fine for now, but it would be nice to have a different workflow in the future.